portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,18 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Apoorva Dhanala's Portfolio</h1>
+      <p>Hi! My name is Apoorva Dhanala and I am a rising junior at the University of Georgia studying Computer Science with an emphasis in Artificial Intelligence. At UGA, I am founder and president of UGA’s chapter of Girls Who Code where we work with local, underprivileged Athens-Clarke County schools where UGA students volunteer weekly hoping to expose students to coding while closing the gender gap in technology. I am also involved with the Georgia Recruitment Team where I represent the admissions office as an ambassador at panels and tours, Athens Prison Tutorial where I tutor prison inmates at the Athens-Clarke County Jail who lack their high school diplomas and help them improve their skills in order to pass their GED examination, and with the Honors Program Student Council where I work to help bring academic and social opportunities to students in the Honors Program. Outside of classes and club involvement, I am a CURO Honors Research scholar where I have been involved with Linguistics and Natural Language Processing research and most recently, Android App development work.  
+</p>
+<p>
+I was born and raised in Atlanta, Georgia and have a little sister. I have learned piano and the classical Indian dance form, Kuchipudi, since I was 5 years old; the skills I have learnt from these art forms, from patience and perseverance to responsibility and delicateness, are a big part of who I am today. Some of my favorite things to do are travel, take pictures, cook, and watch TV. I love traveling to new places and getting to see different parts of the world and I also love photography and taking pictures whether they are travel pictures or senior pictures. As for cooking and baking, some of my favorite things to experiment with are different types of pasta & sauces and also chocolate chip cookies! Whenever I watch TV, some of my go-to’s are Suits, Modern Family or Scandal. 
+ 
+Click on any of the links below to find out more about who I am whether it has to do with my professional experiences or more about my interests and hobbies! 
```

Comment:
Is this the start of a new paragraph? Do you need a  new <p> here?

Reply:
Yeah I chose to make it a start of a new paragraph as I wanted it to start on a new line and be separated from all the content I wrote about.

Reply:
This would be the "Click on any of the links..." part, right?

You'll want to wrap that in a new < p > block to get the new break.

Reply:
I did do that if you look at my index.html file - not sure why its showing like this here.

Reply:
I see! It's showing as an outdated diff. Looks good, thank you. :)